app/admin/AdminLayoutClient.tsx

```diff
@@ -36,6 +36,16 @@ export default function AdminLayoutClient({
           </div>
           <nav>
             <ul className="space-y-2">
+              <li>
+                <Link
+                  href="/"
```

Comment:
Take it out of the menu and side panel and put it in the admin index page directly